src/twisted/mail/pop3.py

```diff
@@ -1534,7 +1557,7 @@ def _dispatch(self, command, default, *args):
         Command X is dispatched to handle_X() if it exists.  If not, it is
         dispatched to the default handler.
 
-        @type command: L{bytes}
+        @type command: L{str}
```

Comment:
I actually expect this to be `bytes`. What lead you to change it?

Reply:
Good point